keps/sig-node/2371-cri-pod-container-stats/README.md

```diff
@@ -272,6 +273,28 @@ Thus, this KEP largely the plan described [here](#plan), with some changes:
 
 #### Open Questions
 1. For the newly introduced CRI API fields, should there be Windows and Linux specific fields?
+    For the alpha implementation, [PR #102789](https://github.com/kubernetes/kubernetes/pull/102789) added a Linux specific field `LinuxPodSandboxStats`. It also added Windows Specific field called  `WindowsPodSandboxStats` but left it blank.  
+
+    The proposal for Widnows support is to add a field `PodSandboxStats` which will be used by both Linux and Windows.  Windows and
+    Linux would both use this field filling in stats that make most sense for them.  Windows is currently missing several stats but can reasonably fill in many of the missing fields (see the [Windows](#windows) section below).  This would be similar approach that is use today for the current CRI `ListContainerStats` call and this would make logic in kubelet more straight forward.
+
+    The field `LinuxPodSandboxStats` would stay the same (with duplicated fields) for backwards compatibility.  We will also leave `WindowsPodSandboxStats` with a comment that it is not currently used.   This option allows us to add specific fields in the future, Linux and Windows sub fields could be added to `LinuxPodSandboxStats` and `WindowsPodSandboxStats` if there are use cases for specific fields.  
+
+    The downside to this approach is that we have two fields (`LinuxPodSandboxStats` and `PodSandboxStats`) which are initially identical. We could mark the individual fields in the `LinuxPodSandboxStats` as depreciated and allow only new linux only fields be added to `LinuxPodSandboxStats`.
```

Comment:
I think this approach makes sense. I think we should mention that eventually we will drop `LinuxPodSandboxStats` and `WindowsPodSandboxStats` in favor of the single structure

Reply:
sounds good, I was thinking that we could possibly use the fields in the future for OS specific stats but probably better to handle at a later point _if_ it ever comes up in the future.

Reply:
updated